### PR TITLE
fix Scale9Sprite draw extra pixels in spritesheet

### PR DESCRIPTION
**Pull request by zilongshanren**
Fixed https://github.com/cocos2d/cocos2d-x/issues/13564

### CODE REVIEW

File: cocos/ui/UIScale9Sprite.cpp [hunk 1]
```diff
@@ -794,7 +794,7 @@ namespace ui {
         //shrink the image size when it is 9-patch
         if(_isPatch9)
         {
-            float offset = 1.4f;
```
Comment (pandamicro) [inline cocos/ui/UIScale9Sprite.cpp:797]: why ?

Comment (WenhaiLin) [inline cocos/ui/UIScale9Sprite.cpp]: @zilongshanren I think it mean one pixel, right?

Comment (zilongshanren) [inline cocos/ui/UIScale9Sprite.cpp:797]: @pandamicro  Android 9-patch image has extra 1 pixel should be removed.
The previous 1.4 is a magic number.

**Comment by zilongshanren** [comment]
[ci rebuild]

**Pull request merged**